interview-questions.md

```diff
@@ -8,48 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a melding of JavaScript syntax and HTML syntax, that lets us use both in the same file.  When using one or the other there are slight syntax convention differences in certain situations as a way to reduce conflict within the language.  One of these differences is `class`, which is HTML syntax for assigning a class, and `className` which is the JavaScript syntax for doing the same thing.  `class` is a reserved word in JavaScript, so another had to be chosen.
 
-Researched answer:
+Researched answer: JSX converts HTML tags into React elements. Though I already knew this, I suppose it would have been helpful to add: using `{}` allows us to write javascript code inside HTML elements.  All code that goes into JSX is intended to be transformed into JavaScript.
 
-1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
+2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is a package management tool for React. it can be used to install and run things like the "Jest" debugging software.
 
-Researched answer:
+Researched answer: Yarn is allows you to share code with others.
```

Comment:
Yarn will modify the yarn.lock file, and may also modify the package.json file.

Reply:
Understood. I think I need to read up more on yarn in general.  Thank you 🙂